mlflow/utils/uri.py

```diff
@@ -72,11 +73,14 @@ def is_databricks_uri(uri):
     return scheme == "databricks" or uri == "databricks"
 
 
-def is_fuse_uri(uri):
+def is_fuse_or_uc_volumes_uri(uri):
     """
-    Validates whether a provided URI is directed to a FUSE mount point
+    Validates whether a provided URI is directed to a FUSE mount point or a UC volumes mount point
     """
-    return any(uri.startswith(x) for x in [_DBFS_FUSE_PREFIX, _DBFS_HDFS_URI_PREFIX])
+    return any(
+        uri.startswith(x)
+        for x in [_DBFS_FUSE_PREFIX, _DBFS_HDFS_URI_PREFIX, _UC_VOLUMES_URI_PREFIX]
```

Comment:
What if there are multiple slashes?

Reply:
Are there any other ways to reference UC volumes?

Reply:
1. Awesome catch on the multiple slashes. TY!! 
2. Yep! There's an internal one that customers (shouldn't use) but I'm adding it in for safety anyway. It'll be retired in the next few quarters, but in the meantime, it's still a failure mechanism, so I'm adding in an exclusion for it.